### PR TITLE
Feature/3048/add toast message

### PR DESCRIPTION
**Pull request by devPriyaJ**
Targets #3048

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,17034 +1,6 @@
 {
```
Comment (Ayush8923) [inline package-lock.json:1]: Please Revert this file from this PR. This file is not needed.

**Review by Ayush8923 — CHANGES_REQUESTED** [review]
@devPriyaJ Please revert `package-lock.json` file from this PR. This file is not needed.

**Comment by Ayush8923** [comment]
@devPriyaJ Are you working on this PR Feedback?

**Comment by GauravGusain98** [comment]
Due to inactivity closing this PR

**Pull request closed without merging**